Documentation/getting-started.md

```diff
@@ -0,0 +1,195 @@
+# Getting started
+
+In this tutorial, we'll show how to use terraform with `matchbox` to provision Container Linux machines.
+
+You'll install the `matchbox` service, setup a PXE network boot environment, and then use terraform configs to describe your infrastructure and the terraform CLI to create those resources on `matchbox`.
+
+## matchbox
+
+Install `matchbox` on a dedicated server or Kubernetes cluster. Generate TLS credentials and enable the gRPC API as directed. Save the `ca.crt`, `client.crt`, and `client.key` on your local machine (e.g. `~/.matchbox`).
+
+* Installing on [CoreOS / Linux distros](deployment.md)
+* Installing on [Kubernetes](deployment.md#kubernetes)
+* Running with [rkt](deployment.md#rkt) / [docker](deployment.md#docker)
+
+Verify the matchbox read-only HTTP endpoints are accessible.
+
+```sh
+$ curl http://matchbox.example.com:8080
+matchbox
+```
+
+Verify your TLS client certificate and key can be used to access the gRPC API.
+
+```sh
+$ openssl s_client -connect matchbox.example.com:8081 -CAfile ~/.matchbox/ca.crt -cert ~/.matchbox/client.crt -key ~/.matchbox/client.key
+```
+
+## Terraform
+
+Install [Terraform](https://www.terraform.io/downloads.html) v0.9.2 on your system.
```

Comment:
I'll change it, but my opinion is this makes docs harder to maintain bc we end up with a bunch of links at the bottom and its unclear which are used/unused. cc @joshix